streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -55,11 +55,7 @@
 
 import java.lang.reflect.Array;
 import java.time.Duration;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
nit: we try to avoid wildcard imports.

Reply:
Gotcha, gotta import the right style guide into IntelliJ :)